de-lib/src/main/java/org/iplantc/de/client/services/UserSessionServiceFacade.java

```diff
@@ -21,8 +22,10 @@
 
     void postClientNotification(JSONObject notification, AsyncCallback<String> callback);
```

Comment:
The Data window is using this callback with the JSONObject and I didn't want to do that refactor in this PR.